.github/workflows/scripts/createTestEnvironments.sh

```diff
@@ -57,7 +58,7 @@ createEnvWithQVCI() {
   echo "teradata-server-name-qvci=$TERADATA_SERVER_NAME" >> $GITHUB_OUTPUT
 }
 
-background createRegularEnv
-background createEnvWithQVCI
+createRegularEnv
```

Comment:
`background` command puts the function invocation in a separate process. This allows `createReuglarEnv` and `createEnvWithQVCI` to proceed in parallel. When you remove `background`, you change it to run sequentially, thus making it run longer. I'm not sure what we gain by running it in sequence?

Reply:
Correct! Adding `background ` back.